Quality_control.do

```diff
@@ -14,31 +16,33 @@
     }
 	
 	***for variables generated from 4_sexual_health
-	foreach var of var w_CPR w_unmet_fp	w_need_fp w_metany_fp w_metmod_fp w_metany_fp_q {
+	foreach var of var w_CPR w_unmet_fp	w_need_fp w_metany_fp w_metmod_fp w_metany_fp_q w_condom_conc {
 	egen pop_`var' = wtmean(`var'), weight(w_sampleweight)
 	}
 	
 	***for variables generated from 7_child_vaccination
 	foreach var of var c_bcg c_dpt1 c_dpt2 c_dpt3 c_fullimm c_measles ///
 	c_polio1 c_polio2 c_polio3{
-    egen pop_`var' = wtmean(`var'), weight(w_sampleweight)
+    egen pop_`var' = wtmean(`var'), weight(hh_sampleweight)
     }
 	
 	***for variables generated from 8_child_illness	
 	foreach var of var c_ari* c_diarrhea 	c_diarrhea_hmf	c_diarrhea_medfor	c_diarrhea_mof	c_diarrhea_pro	c_diarrheaact ///
 	c_diarrheaact_q	 c_illness* c_illtreat* c_sevdiarrhea	c_sevdiarrheatreat ///
 	c_sevdiarrheatreat_q	c_treatAR* c_treatdiarrhea	c_diarrhea_med {
-    egen pop_`var' = wtmean(`var'), weight(w_sampleweight)
+    egen pop_`var' = wtmean(`var'), weight(hh_sampleweight)
     }
 	
 	***for variables generated from 9_child_anthropometrics
-	foreach var of var c_underweight c_stunted	c_hfa c_wfa c_wfh {
-    egen pop_`var' = wtmean(`var'),weight(w_sampleweight)
+	foreach var of var c_underweight c_stunted c_height c_underweight_sev ///
+	c_wasted c_wasted_sev c_weight c_hfa c_wfa c_wfh c_stunted_sev {
+    egen pop_`var' = wtmean(`var'),weight(hh_sampleweight)
     }
 	
 	***for variables generated from 10_child_mortality
-	foreach var of var mor_ali {
-    egen pop_`var' = wtmean(`var'), weight(w_sampleweight)
+	foreach var of var hm_dob hm_doi mor_ade mor_afl mor_ali mor_bord ///
+	mor_dob mor_int mor_male mor_wght mor_wln {
```

Comment:
Thanks for adding these indicators. Most of them are good except two:
mor_wln is not an indicator but the line number for women, mor_dob is not a indicator but the child date at birth.

Reply:
i will remove these two variables.